src/Program.ts

```diff
@@ -154,7 +168,7 @@ export default class Program
                                         ChildProcess.exec(Format('bash -c \'xdg-open "{0}"\'', destination));
                                         break;
                                     default:
-                                        window.showWarningMessage(ResourceManager.Resources.Get("UnsupportetPlatformException"));
+                                        window.showWarningMessage(ResourceManager.Resources.Get("UnsupportedPlatformException"));

```

Comment:
Oh jeez - my bad xDD

Reply:
I btw just noticed I didn't even have this entry to the `/Resources/MarkdownConverter.json`-file.
But nvm I'll add it during the merge-process by myself ^^